showpass/Showpass-E2E-tests
# TA-103 Verify organization contact details modal window and element appearance

**Description:**
This script will be verifying the 'Contact Details' modal window and element appearance.

Steps:

- Navigate to [Test Automation event 1 (DO NOT DELETE) - 101 9 Ave SW - Calgary - Jul 21, 2023](https://beta.showpass.com/ta-event-1/)
- Click the ‘QA team organization' link under the event name
- Make sure the organization ‘Contact Details’ page is correct

**Jira ticket:**
https://showpass.atlassian.net/browse/TA-103

**How to test:**
Run the following spec and ensure all tests pass: "cypress/e2e/home/home-page-appearance.cy.js" 

**Test execution results:**
[Cypress Cloud test results](https://cloud.cypress.io/projects/e9axi7/runs/189/test-results?actions=%5B%5D&browsers=%5B%5D&groups=%5B%5D&isFlaky=%5B%5D&modificationDateRange=%7B%22startDate%22%3A%221970-01-01%22%2C%22endDate%22%3A%222038-01-19%22%7D&orderBy=EXECUTION_ORDER&oses=%5B%5D&specs=%5B%5D&statuses=%5B%5D&testingTypesEnum=%5B%5D)
